fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -201,6 +201,11 @@ class FullyShardedDataParallel(nn.Module):
             GPU OOM during the forward pass. Setting this flag to true will help clearing this
             cache as inner FSDP instances finish part of the forward pass to save GPU memory.
             Default: False
+        force_input_to_fp32 (bool):
```

Comment:
noob, but what if the inputs are not float to begin with ?

Reply:
(I can see that it's actually handled in the conversion code and would not crash, but I'm wondering whether users would be a bit confused)

Reply:
Good point! I will update the docstring to say that only float type of tensor are affected.